docs/source/release_notes.rst

```diff
@@ -6,13 +6,14 @@ Release Notes
     * Enhancements
     * Fixes
     * Changes
+        * Move ``query_by_values`` method from ``Entity`` to ``EntitySet`` (:pr:`1251`)
```

Comment:
Let's also add a "Breaking Changes" section to highlight the move

Reply:
Added